### PR TITLE
Rwahs 476/add representation display

### PR DESCRIPTION
**Pull request by kehh**
goes with https://github.com/rwahs/providence-configuration/pull/36
Includes #66

### CODE REVIEW

File: profile/rwahs.xml [hunk 1]
```diff
@@ -7849,7 +7851,7 @@
             <setting name="remove_first_items"><![CDATA[1]]></setting>
             <setting name="hierarchy_order"><![CDATA[ASC]]></setting>
             <setting name="hierarchy_limit"><![CDATA[0]]></setting>
-            <setting name="hierarchical_delimiter"><![CDATA[ &rArr; ]]></setting>
+            <setting name="hierarchical_delimiter"><![CDATA[ => ]]></setting>
```
Comment (leftclickben) [inline profile/rwahs.xml]: Should this be the unicode character for `&rArr;`, i.e. ⇒ ?

Comment (leftclickben) [inline profile/rwahs.xml]: Or actually, I think the CA default one is `&rarr;`, i.e. →

**Comment by leftclickben** [comment]
Just one question about the delimiter.

File: profile/rwahs.xml (hunk 1 above)
Comment (kehh) [inline profile/rwahs.xml]: The dompdf library shipped with CA apparently only supports some *special* characters (*special special*?). As this report is specifically a printed one, this makes the report fairly legible.

Comment (kehh) [inline profile/rwahs.xml]: (Currently it's outputting `?` in place of ⇒

**Pull request merged**